flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/MeasurementInfoProvider.java

```diff
@@ -49,10 +50,13 @@ public MeasurementInfo getMetricInfo(String metricName, MetricGroup group) {
 
     private static Map<String, String> getTags(MetricGroup group) {
         // Keys are surrounded by brackets: remove them, transforming "<name>" to "name".
-        Map<String, String> tags = new HashMap<>();
-        for (Map.Entry<String, String> variable : group.getAllVariables().entrySet()) {
+        Map<String, String> variables = group.getAllVariables();
+        Map<String, String> tags = new HashMap<>(variables.size() * 4 / 3 + 1);
+        for (Map.Entry<String, String> variable : variables.entrySet()) {
```

Comment:
I would revert this; setting the default size of maps is the kind of obtuse micro optimizations that we don't do.

Reply:
ok